lib/persistence/abstract.js

```diff
@@ -138,6 +138,22 @@ AbstractPersistence.prototype.wire = function(server) {
   };
 };
 
+function arrayFrom(object) {
+  return [].slice.call(object);
+}
```

Comment:
this will be terribly slow. Allocate a new array based on object.length and then add the stuff in there.